ember-cli-import.js

```diff
@@ -57,7 +58,11 @@ CliImport.prototype.bowerDevProd = function(dep, options) {
 CliImport.prototype.bowerFont = function(font, options) {
   var extensions = ['eot', 'svg', 'ttf', 'woff', 'woff2'];
   _.forEach(extensions, function(extension) {
-    this.bower(font + '.' + extension, options);
+    try {
+      fs.accessSync(this.app.bowerDirectory + font + '.' + extension, fs.R_OK)
+      this.bower(font + '.' + extension, options);
+    }
+    catch(err) {}
```

Comment:
Might be worth warning the user that there is a file, but they don't have access to read it (based on your `fs.R_OK` mode check).